pipeline/types/notification.py

```diff
@@ -8,6 +9,10 @@
 class NotificationInput:
     # relevant chat data between the users in the given match
     chatdata: ChatData
+    # datetime to inform Notifier class when pipeline is being run
+    run_datetime: datetime.datetime
+    # the type of notification
+    notification_type: str = ""
 
 
 @dataclass
```

Comment:
Since we are splitting up the data and presentation of Notifications, you can remove `title` and `content` from the `NotificationInfo` class.

Instead, you can add any custom metadata fields you want for the new message reminder. Like the names of the users who messaged you.

Reply:
That sounds good. Although besides names of the users who messsaged, I really can't think of any other relevant metadata. If you, or even the other interns, have any other ideas for fields I'll gladly add them refactor the code accordingly

Reply:
This looks good! Except, notification_type should be a field of NotificationInfo, not NotificationInput.

And as other fields are needed here, we can add them!